website/static/js/fileBrowser.js

```diff
@@ -0,0 +1,799 @@
+            self.newCollectionName('');
+            m.redraw(true);
+            self.dismissModal();
+        };
+        self.deleteCollection = function(){
+            var url = args.collectionMenuObject().item.data.node.links.self;
+            m.request({method : 'DELETE', url : url, config : xhrconfig}).then(function(result){
+                console.log(url, result);
+                for ( var i = 0; i < args.collections.length; i++) {
+                    var item = args.collections[i];
+                    if (item.data.node && item.data.node.id === args.collectionMenuObject().item.data.node.id){
+                        args.collections.splice(i, 1);
+                        break;
+                    }
+                }
+            });
+            self.dismissModal();
+        };
+        self.renameCollection = function () {
+            console.log(args.collectionMenuObject());
+            var url = args.collectionMenuObject().item.data.node.links.self;
+            var nodeId = args.collectionMenuObject().item.data.node.id;
+            var title = args.collectionMenuObject().item.renamedLabel;
+            var data = {
+                'data': {
+                    'type': 'collections',
+                    'id':  nodeId,
+                    'attributes': {
+                        'title': title,
+                    }
+                }
+            };
+            m.request({method : 'PATCH', url : url, config : xhrconfig, data : data}).then(function(result){
+                console.log(url, result);
+                args.collectionMenuObject().item.label = title;
+                m.redraw(true);
+            });
+            self.dismissModal();
+        };
+    },
+    view : function (ctrl, args) {
+        var selectedCSS;
+        var submenuTemplate;
+        return m('.fb-collections', [
+            m('h5', [
+                'Collections ',
+                m('i.fa.fa-question-circle.text-muted', {
+                    'data-toggle':  'tooltip',
+                    'title':  'Collections are groups of projects. You can create new collections and add any project you are a collaborator on or a public project.',
+                    'data-placement' : 'bottom'
+                }, ''),
+                m('.pull-right', m('button.btn.btn-xs.btn-success[data-toggle="modal"][data-target="#addColl"]', m('i.fa.fa-plus')))
+            ]),
+            m('ul', [
+                args.collections.map(function(item, index){
+                    if (item.id === args.activeFilter()) {
+                        selectedCSS = 'active';
+                    } else if (item.id === args.collectionMenuObject().item.id) {
+                        selectedCSS = 'bg-color-hover';
+                    } else {
+                        selectedCSS = '';
+                    }
+                    if (!item.data.systemCollection) {
+                        submenuTemplate = m('i.fa.fa-ellipsis-v.pull-right.text-muted.p-xs', {
```

Comment:
the a tag adds views and is not really meaningful here since these are not links. It;s best to add pointer class to these.